scripts/setup/macos/install_viewer.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+set -eu
+
+# Install cbmc-viewer
+
+# Source kani-dependencies to get CBMC_VIEWER_VERSION
+source kani-dependencies
+
+if [ -z "${CBMC_VIEWER_VERSION:-}" ]; then
+  echo "$0: Error: CBMC_VIEWER_VERSION is not specified"
+  exit 1
+fi
+
+set -x
+
+# brew doesn't recognize specific versions of viewer
```

Comment:
Are we likely to care? Should we ask the tap owners to keep pinned versions in the tap?

Reply:
I can't imagine this becoming a problem since viewer isn't updated that often and also `brew install` installs the latest version of viewer anyway. This patch is also just temporary I imagine, till the issue in github-actions gets fixed or till we find a way to add a path to `pip install`. This was just the quickest solution, to get everyone's PR's unblocked by CI.

Reply:
This is the issue that's causing ci to break, I imagine - https://github.com/actions/runner-images/issues/6507.

Reply:
Can we push the fix to unblock us and we can figure out a better long term plan later? If https://github.com/actions/runner-images/issues/6507 gets fixed, we could probably just revert these changes.